account_oss_report/models/account_oss_report.py

```diff
@@ -0,0 +1,75 @@
+from odoo import api, fields, models
+
+class AccountOssReport(models.Model):
+    _name = "account.oss.report"
+    _description = "OSS Tax Report"
+
+    country_code = fields.Char()
+    tax_rate = fields.Float()
+    base_amount = fields.Float()
+    tax_amount = fields.Float()
+    net_amount = fields.Float()
+    # tax_type = fields.Selection()
+
+    @api.model
+    def _get_account_move_data(self, data=None, start_date=False, end_date=False):
+        """
+        Fetch data from account.move.line with the appropriate domain and calculations.
+        """
+        if data is None:
+            data = []
+
+        # Fetch the tag for "OSS" to use in the domain
+        oss_tag = self.env['account.account.tag'].search([('name', '=', 'OSS')], limit=1)
+
+        # Build the domain to filter by date, tax, and tag "OSS"
+        domain = [
+            ('move_id.invoice_date', '>=', start_date),
+            ('move_id.invoice_date', '<=', end_date),
+            ('tax_ids', '!=', False),
+            ('tax_tag_ids', 'in', [oss_tag.id])
+        ]
+
+        # Fetch move lines that match the criteria
+        move_lines = self.env['account.move.line'].search(domain)
+
+        for line in move_lines:
```

Comment:
The data should be grouped by country code!

And you have to distinct between tax included and excluded.